CMakeLists.txt

```diff
@@ -1,11 +1,9 @@
-cmake_minimum_required(VERSION 2.8.7)
+cmake_minimum_required(VERSION 3.0.2)
```

Comment:
It's grayed out for me too. The tooltip says "The ... branch was force-pushed or recreated".